src/config/network-filesystems.md

```diff
@@ -0,0 +1,70 @@
+# Network Filesystems
+
```

Comment:
I would have a `## NFS` header and then `### Mounting an NFS Share`, but that's personal preference.

Reply:
Yes, that's what i'd like, so that we can later add `## Samba`.